addons/reviewstack/loader.mjs

```diff
@@ -0,0 +1,21 @@
+// Found here: https://stackoverflow.com/a/74931879
+import {existsSync} from 'fs';
+import {basename, dirname, extname, join} from 'path'
+import {fileURLToPath} from 'url'
+
+let extensions = ['mjs', 'js', 'json'], resolveDirs = false
+
+let indexFiles = resolveDirs ? extensions.map(e => `index.${e}`) : []
+let postfixes = extensions.map(e => `.${e}`).concat(indexFiles.map(p => `/${p}`))
+let findPostfix = (specifier, context) => (specifier.endsWith('/') ? indexFiles : postfixes).find(p =>
```

Comment:
This doesn't seem like it will work on Windows?

Reply:
Just realized that same Stackoverflow answer this is from made a package that works on windows too - https://www.npmjs.com/package/extensionless ; still same test plan of running. 

`yarn` in `/addons` folder
`yarn codegen` in `addons/reviewstack`
`yarn start`

Note that `yarn codegen` is running `addons/reviewstack/textmate.js` which also doesn't look like it will run on windows since using unix style paths.